services/backend/services/v3/README.md

```diff
@@ -75,5 +77,6 @@ Here below we show the internal dependencies of the service, which are not alrea
 graph TD   
     rabbitmq --> archivemock
     rabbitmq --> backend
+    ldap --> backend
```

Comment:
Would it make sense, since LDAP is used by both backends, to add a mermaid to the backend/README.md with the services shared by both (it will be useful when adding keycloak)